### PR TITLE
[GeT24Org] add new plugin

### PR DESCRIPTION
**Pull request by oczkers**
Very basic functionality because we're still creating backend.

### CODE REVIEW

**Comment by vuolter** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- module/plugins/hoster/GeT24Org.py  1
- module/plugins/accounts/GeT24Org.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5122031&bid=16838698)

File: module/plugins/hoster/GeT24Org.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+import pycurl
+from hashlib import sha256
```
Comment (vuolter) [inline module/plugins/hoster/GeT24Org.py]: Codacy Issue found: [Unused sha256 imported from hashlib](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5122031)

File: module/plugins/hoster/GeT24Org.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+from ..internal.misc import json
```
Comment (vuolter) [inline module/plugins/hoster/GeT24Org.py]: Codacy Issue found: [Unused json imported from internal.misc](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5122031)

**Comment by GammaC0de** [comment]
Hello,

The naming convention for pyload plugins is to use numbers literally,
So in case of this plugin, the files and `__name__` should be renamed to `GetTwentyFourOrg` instead of `GeT24Org`.

Also, `__version__` should be 0.01, not 0.1.

**Comment by oczkers** [comment]
Sorry i saw Keep2ShareCc with numbers so i thought it's allowed. Anyway it's changed now, is there chance to get write access in future or should i submit pr every time need to chance something?

**Comment by GammaC0de** [comment]
> Sorry i saw Keep2ShareCc with numbers so i thought it's allowed

Well, it is considered "best practices", there may be others that got away but the policy is not use numbers as much as possible.

**Pull request merged**